playbooks/roles/setup-caasp-workers/tasks/apparmor-profiles.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Copy openstack apparmor profile into the workers
+  copy:
+    src: "{{ item }}"
+    dest: "/etc/apparmor.d/local/{{ item }}"
+  with_items: "{{ apparmor_profiles }}"
```

Comment:
Can you use `loop` instead of `with_items` ?

Reply:
done